radixdlt/babylon-wallet-android
# [ABW-3919] Security factors screen redesign

## Description
Please read this short [description](https://radixdlt.atlassian.net/browse/ABW-3917) before review.


## How to test

1. Open wallet and navigate to Security Factors screen
2. Ensure it matches the zeplin designs


## PR submission checklist
- [X] I have tested the Security Factors screen
